tcms/testruns/static/testruns/js/get.js

```diff
@@ -131,12 +131,17 @@ function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
             isAutomatedElement.addClass(isAutomatedIcon)
             isAutomatedElement.attr('title', isAutomatedAttr)
 
-            jsonRPC('TestExecution.get_links', { 'execution_id': testExecution.id, 'is_defect': true }, bugs => {
-                listGroupItem.find('.test-execution-bugs-count').html(bugs.length)
+            jsonRPC('TestExecution.get_links', { 'execution_id': testExecution.id }, links => {
+                const bugCount = links.filter(link => link.is_defect).length;
+                listGroupItem.find('.test-execution-bugs-count').html(bugCount)
+
+                listGroupItem.find('.add-link-button').on('click', () => addLinkToExecution(testExecution))
```

Comment:
This menu item is related directly to the TestExecution row so you don't have to set its handler inside here.

Also use `.click()` for consistency with the rest of the existing code.